packages/project-access/src/project/cap.ts

```diff
@@ -404,8 +403,7 @@ async function loadCdsModuleFromProject(capProjectPath: string): Promise<CdsFaca
     if (global) {
         global.cds = cds;
     }
-    // Ensure we use a known root path, otherwise `cwd` is used which varies between invocations.
-    cds.root = capProjectPath;
```

Comment:
@IainSAP Please review.

Reply:
Should be OK as long as `cds.load(.., capProjectPath)` behaves the same and sets the root internally